src/app/routes/app-routes.tsx

```diff
@@ -77,160 +60,45 @@ export function AppRoutes() {
 }
 
 function useAppRoutes() {
-  const settingsModalRoutes = (
-    <Route>
-      <Route path={RouteUrls.SignOutConfirm} element={<SignOutConfirmDrawer />} />
-      <Route path={RouteUrls.ChangeTheme} element={<ThemesDrawer />} />
-      <Route path={RouteUrls.SelectNetwork} element={<SelectNetwork />} />
-    </Route>
-  );
-
-  const legacyRequestRoutes = (
-    <>
-      <Route
-        path={RouteUrls.TransactionRequest}
-        element={
-          <AccountGate>
-            <Suspense fallback={<SuspenseLoadingSpinner />}>
-              <TransactionRequest />
-            </Suspense>
-          </AccountGate>
-        }
-      >
-        {ledgerStacksTxSigningRoutes}
-        <Route path={RouteUrls.EditNonce} element={<EditNonceDrawer />} />
-        <Route path={RouteUrls.TransactionBroadcastError} element={<BroadcastErrorDrawer />} />
-      </Route>
-      <Route
-        path={RouteUrls.SignatureRequest}
-        element={
-          <AccountGate>
-            <Suspense fallback={<SuspenseLoadingSpinner />}>
-              <StacksMessageSigningRequest />
-            </Suspense>
-          </AccountGate>
-        }
-      >
-        {ledgerStacksMessageSigningRoutes}
-      </Route>
-      <Route
-        path={RouteUrls.ProfileUpdateRequest}
-        element={
-          <AccountGate>
-            <Suspense fallback={<SuspenseLoadingSpinner />}>
-              <ProfileUpdateRequest />
-            </Suspense>
-          </AccountGate>
-        }
-      />
-      <Route
-        path={RouteUrls.PsbtRequest}
-        element={
-          <AccountGate>
-            <Suspense fallback={<SuspenseLoadingSpinner />}>
-              <PsbtRequest />
-            </Suspense>
-          </AccountGate>
-        }
-      />
-    </>
-  );
-
-  const rpcRequestRoutes = (
-    <>
-      <Route
-        path={RouteUrls.RpcGetAddresses}
-        element={
-          <AccountGate>
-            <RpcGetAddresses />
-          </AccountGate>
-        }
-      />
-      {rpcSendTransferRoutes}
-      <Route
-        path={RouteUrls.RpcSignBip322Message}
-        lazy={async () => {
-          const { RpcSignBip322MessageRoute } = await import(
-            '@app/pages/rpc-sign-bip322-message/rpc-sign-bip322-message'
-          );
-          return { Component: RpcSignBip322MessageRoute };
-        }}
-      />
-      <Route
-        path={RouteUrls.RpcSignPsbt}
-        element={
-          <AccountGate>
-            <RpcSignPsbt />
-          </AccountGate>
-        }
-      />
-      <Route
-        path={RouteUrls.RpcSignPsbtSummary}
-        element={
-          <AccountGate>
-            <RpcSignPsbtSummary />
-          </AccountGate>
-        }
-      />
-    </>
-  );
-
   return createHashRouter(
     createRoutesFromElements(
       <Route element={<Container />}>
         <Route
-          path={RouteUrls.Home}
+          path={`/${RouteUrls.Receive}/${RouteUrls.ReceiveStx}`}
```

Comment:
To go direct to `/receive/stx` for example in Ledger mode, we need this declaration also. 

Doing this outside of `<Home />` means we bypass the check for `<Routes location={backgroundLocation || location}>` 

We can't do that check in this file without moving from  `createHashRouter` -> `<HashRouter>` or introducing another wrapper which I can investigate

Reply:
Confused by this. What's ledger mode got to do with it?